### ISSUE TITLE
no iommu kubevirt

### ISSUE THREAD

**Issue by dhruvik7** [jear/my-kubevirt-nvidia#1]
Hi, I see that you are using vfio with the no-iommu unsafe parameter. Were you able to get this working with kubevirt?

**Comment by jear** [comment]
Yes, for worker node with nvidia operator using node label  passthrough, it is working. 

![image](https://github.com/jear/my-kubevirt-nvidia/assets/4311242/e2a16f12-0279-48f7-96a0-3c75c7ef6722)

**Comment by dhruvik7** [comment]
How were you able to get the vm to access the device at /dev/vfio/noiommu-* ? I tried renaming to /dev/vfio/0 based on the naming it expects, but that gives an Operation not permitted error.
